packages/Button/Button.md

```diff
@@ -39,3 +39,13 @@ text is too low in the hover state.
 ```jsx { "props": { "className": "docs_purple-block" } }
 <Button variant="inverted">Get started</Button>
 ```
+
+### Using A11yContent
+
+Use the `A11yContent` component to create invisible text that is read out loud by screen readers.
+
+```jsx
+<Button href="#">
```

Comment:
Can we remove the href="#"? If we are keeping the href, it should be a <ButtonLink />